### PR TITLE
Updated and moved contact-us pages

### LINKED ISSUE #2746: Update and move /about/contact-us
## Summary

1. Move update copy doc per Mark
2. move page to /contact-us
3. make sure navigation works and the actual contact us form.

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Updated and moved contact-us pages

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [contact us](http://0.0.0.0:8001/contact-us), [form](http://0.0.0.0:8001/contact-us/form) and [thank-you]http://0.0.0.0:8001/contact-us/form/thank-you)
- compare to the [copy docs](https://drive.google.com/drive/folders/0Bw_yUXzV8CSHYjJ6VkNuanVWQVE?usp=sharing)

## Issue / Card

Fixes #2746

### CODE REVIEW

File: templates/contact-us/index.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
+{% extends "contact-us/base_contact-us.html" %}
+
+{% block title %}Contact Canonical{% endblock %}
+
```
Comment (Caleb-Ellis) [inline templates/contact-us/index.html]: Do we need the meta description here?
`Ubuntu is an open source software platform that runs everywhere from the smartphone, the tablet and the PC to the server and the cloud.`

Comment (pmahnke) [reply templates/contact-us/index.html]: That's a generic one... and bad as we don't do smartphones and tablets... I added a better one for contact-us and updated the copy doc

**Comment by kwm14** [comment]
## Design review
One minor change 😄 
- add suffix to first row to allow a bit of breathing room between content - `col-6 suffix-1`

screen shot 2018-02-26 at 14 58 19

**Comment by pmahnke** [comment]
@kwm14 ready for re-review

**Comment by kwm14** [comment]
Thank @pmahnke 👍

**Pull request merged**